examples/Photo-z example.ipynb

```diff
@@ -1,326 +0,0 @@
-{
```

Comment:
I'm fine with removing the core functionality, but I don't think we should lose the examples. Since calculating photo-z pdfs is important for correlation function calculations, we should show an alternative way of doing it now that the built-in CCL functions have been removed.